.changeset/twelve-shrimps-jog.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+---
+---
+"section": "tree"
+---
+Implicitly constructed object nodes now only consider own properties during validation.
+
+When determining if some given data is compatible with a particular ObjectNode schema, both inherited and own properties used to be considered.
+However when actually constructing the node from the data, only own properties were used.
+This could lead to input containing inherited properties getting validated, but producing out of schema nodes missing fields.
```

Comment:
```suggestion
This could lead to input containing inherited properties being validated, but producing out-of-schema nodes that were missing fields.
```

Wasn't sure the intended meaning, so made an attempt.